features.js

```diff
@@ -186,65 +186,52 @@ function getCodec(peerConnectionLog, kind, direction) {
     }
 }
 
-// extract a local/remote audio or video track.
-function extractTrack(peerConnectionLog, kind, direction) {
-    var trackId;
-    var reports = [];
-    var streamevent = 'onaddstream';
-    var trackevent = 'ontrack';
-    if (direction === 'send') {
-        streamevent = 'addStream';
-    }
-    // search for the (first) track of that kind.
-    for (var i = 0; i < peerConnectionLog.length; i++) {
-        var type = peerConnectionLog[i].type;
-        if (direction === 'send') {
-            if (type === 'addTrack') {
-                var kindAndTrack = peerConnectionLog[i].value.split(' ')[0].split(':');
-                if (kindAndTrack[0] === kind) {
-                    trackId = kindAndTrack[1];
-                    break;
-                }
```

Comment:
missing else if type === addStream -- or should that have been handled elsewhere? Anyway...